esp/esp/program/modules/handlers/studentregcore.py

```diff
@@ -91,6 +93,7 @@ def students(self, QObject = False):
 
         retVal = {'confirmed': ESPUser.objects.filter(q_confirmed).distinct(),
                   'attended' : ESPUser.objects.filter(q_attended).distinct(),
+                  'checked_out': ESPUser.objects.filter(q_checked_out).distinct(),
```

Comment:
It's giving me 0 students in the AUL even though I have students checked out...

Reply:
Have the students been checked back in since they were checked out? Are they checked out for the program you are using? This worked for me, so I'm not really sure how to reproduce the problem you're having.

Reply:
They are currently checked out for the program on hand (haven't been checked back in)

Reply:
Are the checked out students excluded properly in the attendance module?

Reply:
Yeah they were. But also, I just rechecked the AUL/ grid based class changes in an effort to help you debug and now the checked out students appear properly (removed from grid and added to AUL). When I tested this the other day, I had refreshed the cache & hard refreshed. Are there any other time-based things in place that might have made it not work right away (~5 min after checking out the students)?

Reply:
Umm...it's possible there are supercached things that aren't cleared when you flush the cache and are only updated once you run fab manage:update. Anywho, glad to hear this is resolved, thanks!

Reply:
Actually, can you try checking out a student and seeing if the AUL is updated immediately? (since you have your dev server running now)

Reply:
I just tried checking out a student, and grid based & AUL don't change (even after cache clear & hard refresh) whereas attendance does. Personally I think this is a problem and not resolved? Because grid-based is supposed to be accurate to the minute in terms of class changing students and if you checkout students and they stay there for some unknown period of time, it seems a little useless to me

Reply:
OK, I can update grid-based class changes to use the same code as the attendance module. I'm not sure I can make the AUL update faster, unfortunately.